pipeline/tasks/package_metadata_tasks.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tasks related to package metadata"""
+
+import os
+import yaml
+
+from pipeline.tasks import task_base
+from pipeline.utils import task_utils
+
+
+class PackageMetadataConfigGenTask(task_base.TaskBase):
```

Comment:
This is per the discussion in https://github.com/googleapis/toolkit/issues/780; it's not really doing "generation" -- more just merging some existing configs. If the logic ever needs to extend beyond merging a few existing files, I agree that this should be in toolkit.